doc/userguide/lua/lua-functions.rst

```diff
@@ -6,7 +6,8 @@ Lua functions
 Differences between `output` and `detect`:
 ------------------------------------------
 
-Currently, the ``needs`` key initialization varies, depending on what is the goal of the script: output or detection.
+Currently, the ``needs`` key initialization varies, depending on what is the goal of the script: output or detection. The
+Lua script for the ``luaxform`` transform **does not use ``needs``**.
```

Comment:
PR example does show needs

Reply:
Question: I'll make changes to eliminate the mention of `needs` in the test/doc content.

Should we retain the `init` function and call it during initialization and *not use/check* for a return falue?